### ISSUE TITLE
need help....

### ISSUE THREAD

**Issue by Arshiya051** [dufernst/LegionCore-7.3.5#155]
Hello. I want to complete a quest by clicking on a teleporter game-object. can you help me?
Id object=249741
Id Quest = 41574

This quest has two parts. The first one is teleported by clicking on an NPC and enters the scenario. And at the end of the way, with this object, you will be teleported to Ulduar and the quest must be completed. which cannot.  Now I just need you to help me to compile my quest by clicking on this object.
@dufernst 
@T-R-I-X 
@Fragrus

**Comment by Arshiya051** [comment]
Please give an answer at least.
@dufernst

**Comment by Elgrimm** [comment]
This repository is dead
I am currently working on a new one and will give a link here in the near future.

**Comment by Melanholyx** [comment]
@Elgrimm  waiting for it.  this core is not buildable anymore I guess. because boost 1.64 doesn't work for me. I tried to build with newer boost via Cmake but then I have alot of errors in VS building. if you can help me I would be glad :) p.s waiting for that new link

**Comment by Elgrimm** [comment]
@Melanholyx speak Russia?

**Comment by Melanholyx** [comment]
hello no, only english. fixed Cmake Issue but now have problem with compiling in VS 2017 if you have provide simple help I would be glad :) discord : _imno

**Comment by Arshiya051** [comment]
> This repository is dead I am currently working on a new one and will give a link here in the near future.

Yes, I know this is not scripted. But it can be fixed without writing a script. Now you are working very well. Thank you for this article.
But again, I didn't understand how a quest can be completed by clicking on a game-object teleporter.
@Elgrimm

**Comment by Elgrimm** [comment]
@Arshiya051 Hi! I don't quite understand what you want, but I'll explain how it works. There is a task, for example, with ID 45673 and it has criteria for which the task will be completed. To begin with, if you want to make a killcredit, you need to find the ID of kill credits in the quest_objectives table, and then make a SmartAI for the object. action_type 33

**Comment by Elgrimm** [comment]
Accordingly, in event_type you specify depending on the type of the gossip_hello object

**Comment by Elgrimm** [comment]
taking into account that the object has type 10 in the gameobject_template table

**Comment by Arshiya051** [comment]
> This repository is dead I am currently working on a new one and will give a link here in the near future.

hello Could you complete this scenario (quest)? @Elgrimm
